app/controllers/client_notifications_controller.rb

```diff
@@ -0,0 +1,56 @@
+class ClientNotificationsController < ApplicationController
+  before_action :set_client_notification, only: [:edit, :update, :destroy]
+  before_action :translate_model_name, only: [:update, :destroy]
+
+  def index
+    authorize ClientNotification
+    @client_notifications = ClientNotification.all
+  end
+
+  def new
+    @client_notification = ClientNotification.new
+    authorize @client_notification
+  end
+
+  def edit; end
+
+  def create
+    @client_notification = ClientNotification.new(client_notification_params)
+    @client_notification.user = current_user
+    authorize @client_notification
+    if @client_notification.save
+      redirect_to client_notifications_path,
+        notice: t('crud.created', model: @client_notification.class.model_name.human)
+    else
+      render :new
+    end
+  end
+
+  def update
+    if @client_notification.update(client_notification_params)
+      redirect_to client_notifications_path, notice: t('crud.updated', model: @translated_model_name)
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @client_notification.destroy
+    redirect_to client_notifications_url, notice: t('crud.destroyed', model: @translated_model_name)
```

Comment:
change the method `translate_model_name` and use it here:

```ruby
redirect_to client_notifications_path, notice: t('crud.created', model: translate_model_name)
```

Reply:
PS: Maybe rename `translate_model_name` to something in the manner of `human_model_name` or something else you find explicit.